website/layout.py

```diff
@@ -0,0 +1,65 @@
+from pathlib import Path
+
+import dash
+import dash_bootstrap_components as dbc
+from dash import dcc
+
+from .components import captioned_image_row
+
+
+def content() -> list:
+    """Generate the content for the webpage.
+
+    Returns:
+        list: A list of each component ordered from the top of the page to the bottom.
+    """
+    # Read the static markdown content in static/index.md for the top of the webpage.
+    # Edit this file to edit the text content at the top of the page.
```

Comment:
Just to clarify which file file it is that needs editing. 
```suggestion
    # Edit that file to modify the text content at the top of the page.
```